question-generation/comments-wrangling/webpack.config.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-undef */
+
+const devCerts = require("office-addin-dev-certs");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const urlDev = "https://localhost:3000/";
+const urlProd = "https://www.contoso.com/"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
```

Comment:
This line should have a `TODO` on it (in MS's template) so we don't forget to change it.

Reply:
Will also be reminded in README as well.